WooCommerce/Classes/Yosemite/AuthenticatedState.swift

```diff
@@ -57,6 +57,16 @@ class AuthenticatedState: StoresManagerState {
             TaxClassStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
         ]
 
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.cardPresentPayments) {
+            let cardReaderStore =  CardPresentPaymentStore(dispatcher: dispatcher,
+                                                           storageManager: storageManager,
+                                                           network: network,
+                                                           cardReaderService: ServiceLocator.cardReaderService)
+            allServices.append(cardReaderStore)
+        }
+
+        services = allServices
+
```

Comment:
Why not enable the CardPresentPaymentStore always? Just the functionality is behind a feature flag, but I don't think that we should hide the store behind the flag (in theory it should not be called from anywhere else right)?

Reply:
That's a good point, Paolo. I'll update and merge.